cli/foundation-models/system/inference/image-text-embeddings/image-text-embeddings-batch-endpoint.sh

```diff
@@ -0,0 +1,165 @@
+set -x
+# The commands in this file map to steps in this notebook: https://aka.ms/azureml-infer-batch-sdk-image-classification
+# The sample scoring file available in the same folder as the above notebook.
+
+# script inputs
+registry_name="azureml"
+subscription_id="<SUBSCRIPTION_ID>"
+resource_group_name="<RESOURCE_GROUP>"
+workspace_name="<WORKSPACE_NAME>"
+
+# This is the model from system registry that needs to be deployed
+model_name="image-text-embeddings-openai-clip-vit-base-patch32"
```

Comment:
You probably have it on your radar, but adding it here anyway. Update the model name once changed.